### PR TITLE
Refine styling

### PR DESCRIPTION
**Pull request by jennieramida**
## Describe your changes
- refine style and theme on Button, Tooltip, Badge, Tag and etc.

### CODE REVIEW

File: src/lib/pages/codes/index.tsx [hunk 1]
```diff
@@ -64,16 +64,22 @@ const Codes = observer(() => {
...
   return (
     <PageContainer>
-      <Heading as="h1" size="lg" color="white" pb={8}>
+      <Heading
+        variant="h5"
+        as="h5"
+        minH="36px"
+        display="flex"
+        alignItems="center"
+      >
```
Comment (bkioshn) [inline src/lib/pages/codes/index.tsx:73]: remove color

File: CHANGELOG.md [hunk 2]
```diff
@@ -41,6 +41,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Improvements
 
+- [#311](https://github.com/alleslabs/celatone-frontend/pull/311) Refine css styling
```
Comment (bkioshn) [inline CHANGELOG.md:44]: Merge develop for changelog update

File: src/lib/styles/theme/components/tag.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    },
+    md: {
+      container: {
+        maxH: "24px",
+        fontSize: "12px",
+      },
+    },
+  },
+  variants: {
+    violetLight,
```
Comment (bkioshn) [inline src/lib/styles/theme/components/tag.ts:60]: others variants are using kebab case eg. "outlint-primary" might consider changing this

File: src/lib/components/table/contracts/TagsCell.tsx [hunk 4]
```diff
@@ -88,17 +76,7 @@ export const TagsCell = ({
...
-                    key={item}
-                    borderRadius="full"
-                    variant="solid"
-                    bgColor="honeydew.darker"
-                    color="text.main"
-                  >
-                    <TagLabel>{item}</TagLabel>
-                  </Tag>
-                );
+                return <Tag key={item}>{item}</Tag>;
```
Comment (songwongtp) [inline src/lib/components/table/contracts/TagsCell.tsx:79]: `tags.map((item) => (<Tag key={item}>{item}</Tag>))`

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/pages/proposals/index.tsx [hunk 5]
```diff
@@ -10,6 +10,7 @@ import InputWithIcon from "lib/components/InputWithIcon";
 import PageContainer from "lib/components/PageContainer";
 import { Pagination } from "lib/components/pagination";
 import { usePaginator } from "lib/components/pagination/usePaginator";
+import { Tooltip } from "lib/components/TooltipComponent";
```
Comment (evilpeach) [inline src/lib/pages/proposals/index.tsx:13]: I'd rather call it 'Tooltip' than 'TooptipComponent'.

File: src/lib/components/table/EditableCell.tsx [hunk 6]
```diff
@@ -194,13 +188,7 @@ export const EditableCell = ({
...
             )}
             {!!tooltip && (
-              <Tooltip
-                hasArrow
-                label={tooltip}
-                placement="top"
-                bg="honeydew.darker"
-                arrowSize={8}
-              >
+              <Tooltip hasArrow label={tooltip}>
```
Comment (evilpeach) [inline src/lib/components/table/EditableCell.tsx:191]: remove `hasArrow`

File: src/lib/components/button/InstantiateButton.tsx [hunk 7]
```diff
@@ -81,13 +82,7 @@ export const InstantiateButton = ({
...
 
   return (
-    <Tooltip
-      hasArrow
-      label={tooltipLabel}
-      placement="top"
-      arrowSize={8}
-      bg="honeydew.darker"
-    >
+    <Tooltip hasArrow label={tooltipLabel}>
```
Comment (evilpeach) [inline src/lib/components/button/InstantiateButton.tsx:85]: remove `hasArrow` ?

**Review by evilpeach — APPROVED** [review]

**Pull request merged**